docs/directives/parse-as-csv.md

```diff
@@ -0,0 +1,52 @@
+# Parse as CSV
+
+The `parse-as-csv` is a directive for parsing an input record as comma-separated values.
+
+
+## Syntax
+```
+parse-as-csv <column-name> <delimiter> <skip-on-error>
```

Comment:
I think skip-on-error is actually parse headers now. parse-as-csv <column> <delimiter> [<header=true/false>]